FAQ.md

```diff
@@ -0,0 +1,71 @@
+
+kcp has a concept call [Syncer](https://github.com/kcp-dev/kcp/blob/main/docs/terminology.md#syncer) which is installed on each [WorkloadClusters](https://github.com/kcp-dev/kcp/blob/main/docs/terminology.md#workload-cluster). The [Syncer](https://github.com/kcp-dev/kcp/blob/main/docs/terminology.md#syncer) negotaties, with kcp, a set of available APIs to make accessible in the workspace. This may include things like [Deployments](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) or other resources you may explicitly configure the syncer to synchronize to kcp. Once these APIs are made available in your [Workspace](https://github.com/kcp-dev/kcp/blob/main/docs/terminology.md#workspace) you may then create resources of that type. From there, the [Location and Placement](https://github.com/kcp-dev/kcp/blob/main/docs/terminology.md#location) APIs help determine which [Location](https://github.com/kcp-dev/kcp/blob/main/docs/terminology.md#location) your deployable resource lands on.
+
+## Will KCP be able to pass the K8S conformance tests in [CNCF Conformance Suites](https://www.cncf.io/certification/software-conformance/)?
+
+For supported APIs, kcp is Kubernetes and will be compliant with conformance suites.  However, not all APIs are supported out of the box, on purpose.
+
+## Are these ideas being presented to the Kubernetes community?
+
+Yes! All development is public and we have started discussions about what is mature enough to present to various interested parties. It is worth noting that not all of the goals of kcp are necessarily part of the goals of Kubernetes so while some items may be accepted upstream we expect that at least some of the concepts in kcp will live outside the Kuberenetes repository.
+
+## How does upgrading Kubernetes work in this model?
+
+kcp depends on a [fork](https://github.com/kcp-dev/kubernetes) of Kubernetes. Updating Kuberentes for the kcp project itself requires a rebase. We are actively following the releases of Kubernetes and rebasing regularly. Updating Kuberentes for clusters attached to kcp is exactly like it is done today, though you may choose to follow different patterns of availability for applications based on kcp's ability to cordon and drain clusters or relocate applications.
+
+## Can kcp workloads run on MiniKube clusters?
+
+Yes.
+
+## How will storage work at the workspace level for administration?
+
+We are in the early stages of (brainstorming storage use cases)[https://docs.google.com/document/d/13VpnyBQHpaishrastdO3kGApKLzYBeb9QXdKn3o2vHs/edit#heading=h.tg51mxx1tg19]. Please join in the conversation if you have opinions or use cases in this areas.
+
+## With multiple workspaces on a single cluster, that implies pods from multiple tenants are on the same host VM. Does this mean privileged pods are forbidden to avoid cross contamination with host ports and host paths?
```

Comment:
```suggestion
## With multiple `Workspaces` on a single cluster, that implies `Pods` from multiple tenants are on the same host VM. Does this mean privileged `Pods` are forbidden to avoid cross contamination with host ports and host paths?
```

(if we want to be formal + consistent with terminology)